prime-angular-frontend/src/app/shared/components/forms/college-certification-form/college-certification-form.component.ts

```diff
@@ -67,7 +67,13 @@ export class CollegeCertificationFormComponent implements OnInit {
     private enrolmentService: EnrolmentService
   ) {
     this.remove = new EventEmitter<number>();
-    this.licenses = this.configService.licenses.filter(l => l.collegeLicenses.filter(cl => cl.discontinued).length === 0);
```

Comment:
I don't understand how previous code filtered out this licence?
```
            {
                "code": 9,
                "name": "College of Dietitians of BC",
                "weight": 70,
                "collegeLicenses": [
                    {
                        "collegeCode": 9,
                        "licenseCode": 64,
                        "collegeLicenseGroupingCode": null,
                        "discontinued": false
                    }
                ],
                "collegePractices": []
            },
```

Reply:
Each license object contains a list of CollegeLicense objects. Some of the CollegeLicense objects have the 'discontinued' property set to true. The previous code filtered out the licenses objects with any CollegeLIcense 'discontinued' property being true. The license code 64 happens to have one CollegeLicense object is set discontinued and the license code 64 is removed from the license list.

Reply:
Oh, I was looking at the wrong JSON, should be the following I think  

```
            {
                "code": 64,
                "name": "Not Displayed",
                "weight": 1,
                "prefix": "",
                "manual": true,
                "validate": false,
                "namedInImReg": false,
                "licensedToProvideCare": false,
                "allowRequestRemoteAccess": false,
                "prescriberIdType": null,
                "collegeLicenses": [
                    {
                        "collegeCode": 4,
                        "licenseCode": 64,
                        "collegeLicenseGroupingCode": null,
                        "discontinued": false
                    },
                    {
                        "collegeCode": 5,
                        "licenseCode": 64,
                        "collegeLicenseGroupingCode": null,
                        "discontinued": true
                    },
                    {
                        "collegeCode": 6,
                        "licenseCode": 64,
                        "collegeLicenseGroupingCode": null,
                        "discontinued": true
                    },
...
```